src/components/Footer/Footer.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import Footer from './Footer';
+
+storiesOf('Footer', module).add('Footer', () => (
+  <div>
+    <Footer />
+  </div>
+));
```

Comment:
Structure this file in same way as other storybook, e.g.

```javascript
const FooterExample = () => (
  ...footerMarkup
)

const stories = storiesOf('Components|Footer');

stories.add(....);
```

See https://github.com/wellcometrust/corporate-components/blob/develop/src/components/Section/Section.stories.tsx as an example.

Reply:
haven't done stories yet. This is just a file created by create folder command.

Reply:
Fair enough - if you can tidy this up anyway, we can turn this in to an actual (not draft) PR this afternoon and then it means any of us can build individual Footer components.

e.g. I'm working on the Newsletter Signup, but I can't put it it in the Footer til it exists!